aiida/orm/importexport.py

```diff
@@ -729,7 +761,57 @@ def import_data_dj(in_path, user_group=None, ignore_unknown_nodes=False,
                 for import_entry_id, entry_data in existing_entries[model_name].items():
                     unique_id = entry_data[unique_identifier]
                     existing_entry_id = foreign_ids_reverse_mappings[model_name][unique_id]
+                    import_data = dict(deserialize_field(
+                        k, v, fields_info=fields_info,
+                        import_unique_ids_mappings=import_unique_ids_mappings,
+                        foreign_ids_reverse_mappings=foreign_ids_reverse_mappings)
+                                       for k, v in entry_data.items())
                     # TODO COMPARE, AND COMPARE ATTRIBUTES
+
```

Comment:
This code has very few things that are backend dependent - it can go easily to a function that can be shared by both backend import functions

Reply:
I partly agree - although there are some potential db entry deletions.
Should I try to create a common function or leave it as it?